### DISCUSSION TITLE
维护的镜像是不是有什么误解？？？https://hub.docker.com/r/ddsderek/mw

### DISCUSSION THREAD

**Discussion by taurusduan** [midoks/mdserver-web#373] [Ideas]
刚才下载了一下镜像，发现整个页面都没有安全地址。。。确实很安全。。。

个人说一下自己的想法，没有恶意。我个人觉得镜像就是为了快速一键部署而生的，所以你提供账号密码，不提供登陆安全地址，到底想要表达什么呢？难道使用的人不会修改吗？还是说你压根就忘记了？理解不了。。。

**Comment by Wittoy** [comment]
这玩意可以后续设置吧...容器这玩意都是打包好的，就算设置好安全地址也是固定的

**Comment by midoks** [comment]
可以打包docker，本人不维护，别人维护
